src/main/java/io/vertx/core/eventbus/impl/EventBusImpl.java

```diff
@@ -400,6 +400,10 @@ protected String generateReplyAddress() {
     sendOrPubInternal(newSendContext(message, options, handler, writePromise));
   }
 
+  public boolean isAddressLocal(String address) {
```

Comment:
This seemed to be necessary to support the hybrid use case when addresses are published consumed locally only in some verticals but global in other others.